cmp/views.py

```diff
@@ -1,5 +1,8 @@
 from django.shortcuts import render, HttpResponse
 
+from IPython import embed
```

Comment:
once we install ipdb we won't need to use embed and can use breakpoint() which is standard python.